lib/ipmi_vita.c

```diff
@@ -202,11 +171,9 @@ vita_discover(struct ipmi_intf *intf)
 		lprintf(LOG_INFO, "Invalid completion code received: %s",
 			val2str(rsp->ccode, completion_code_vals));
 	} else if (rsp->data_len < 5) {
-		lprintf(LOG_INFO, "Invalid response length %d",
-			rsp->data_len);
+		lprintf(LOG_INFO, "Invalid response length %d", rsp->data_len);
```

Comment:
clang-format tries to break as late as it can for parameters such that, in the top case it couldn't have the format string on the same line as the log level.

Reply:
In this case, it seems a bit wonky since it could have fit that line.